e2e/rbd_helper.go

```diff
@@ -59,15 +59,19 @@ func createRBDStorageClass(
 	sc.Parameters["csi.storage.k8s.io/node-stage-secret-namespace"] = cephCSINamespace
 	sc.Parameters["csi.storage.k8s.io/node-stage-secret-name"] = rbdNodePluginSecretName
 
-	fsID, stdErr, err := execCommandInToolBoxPod(f, "ceph fsid", rookNamespace)
-	if err != nil {
-		return err
-	}
-	if stdErr != "" {
-		return fmt.Errorf("error getting fsid %v", stdErr)
+	fsID := defaultFSID
+	if fsID == noFSID {
+		var stdErr string
+		fsID, stdErr, err = execCommandInToolBoxPod(f, "ceph fsid", rookNamespace)
+		if err != nil {
+			return err
+		}
+		if stdErr != "" {
+			return fmt.Errorf("error getting fsid %v", stdErr)
+		}
+		// remove new line present in fsID
+		fsID = strings.Trim(fsID, "\n")
```

Comment:
Should we trim regardless of whether or not FSID is passed?

Reply:
I don't think that is needed, users will run `--fsid=openshift-storage` or similar, that usually would not contain additional spaces